app/views/pages/index.html.erb

```diff
@@ -1,9 +1,30 @@
-<div class="btn-group pull-right" data-toggle="buttons">
-    <% Filter.list.each do |filter| %>
-        <label id="<%= filter.filter_id %>" class="btn btn-primary" onclick="localStorage.setItem('jobfilter', '<%= filter.filter_id %>'); window.location = '?jobfilter=<%= filter.filter_id %>';" >
-          <input type="radio" name="<%= filter.filter_id %>" id="option_user" > <%= filter.title %>
-        </label>
-    <% end %>
+<div class="row pull-right">
+  <div class="div-inline dropdown">
```

Comment:
Could you get away with using the documented Bootstrap way for creating button dropdowns?

http://getbootstrap.com/components/#btn-dropdowns-single

```html
<div class="btn-group">
  <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
    Action <span class="caret"></span>
  </button>
  <ul class="dropdown-menu">
    <li><a href="#">Action</a></li>
    <li><a href="#">Another action</a></li>
    <li><a href="#">Something else here</a></li>
    <li role="separator" class="divider"></li>
    <li><a href="#">Separated link</a></li>
  </ul>
</div>
```